### ISSUE TITLE
Allow previously elected custodians to still see the "My Payments" menu

### ISSUE THREAD

**Issue by lukestokes** [eosdac/eosdac-client-legacy#58]
When custodians are voted out, they lose access to that menu and can no longer claim their pay for the previous week using the member client. This happened to investingwad and jbucksteiner this past week.

**Comment by piecesnbits** [comment]
suggestion?

**Comment by lukestokes** [comment]
Maybe create a page under dac activity which shows all custodian payments which haven't been claimed yet. Then, on that page only, you can do a lookup for the logged in user to see if there is a match. If there is, then show the claim pay button next to that entry. It might also be cool to show a history of claimed payments.

What do you think @piecesnbits ?

**Comment by piecesnbits** [comment]
good idea

**Comment by piecesnbits** [comment]
history of payments will be impossible without history api though
they get removed from the table when paid...

**Comment by lukestokes** [comment]
Yeah, that's fine. If @michaeljyeates wants to add that to the eosdac-api, throw it in mongo, and make it available, that would be cool, but it doesn't seem like a major issue to me. I just want to make sure custodians who get voted out can still get their last week of pay.

**Comment by michaeljyeates** [comment]
It should be possible to just query outstanding payments, dont think it needs an api

**Comment by piecesnbits** [comment]
yeah outstanding payments is possible, history isn't.
